report_xlsx/controllers/main.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2017 Creu Blanca
+# Copyright 2021 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnuorg/licenses/agpl.html).
+
+import json
+import logging
+
+from werkzeug.urls import url_decode
+
+from odoo.http import (
+    content_disposition,
+    request,
+    route,
+    serialize_exception as _serialize_exception,
+)
+from odoo.tools import html_escape
+from odoo.tools.safe_eval import safe_eval, time
+
+from odoo.addons.web.controllers import report
```

Comment:
ReportController should be imported like this  
```
from odoo.addons.web.controllers.report import ReportController
class ReportController(ReportController):
```
because the controllers were reorganized